components/widgets/ui/src/main/resources/Main/MessageSenderMacro.xml

```diff
@@ -25,12 +25,9 @@
   <defaultLanguage/>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
-  <creationDate>1424473682000</creationDate>
```

Comment:
Why not use XWiki's `MessageSenderMacro.xml` entirely? Seems the fix that was the reason of backporting is already included in the release.